CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+## Versions
+
+## x.x.x
+* Add Segment Targeting APIs. For more info, check out our [docs](https://developers.applovin.com/en/max/godot/overview/data-and-keyword-passing/).
+* Add Terms and Privacy Policy Flow APIs. For more info, check out our [docs](https://developers.applovin.com/en/max/godot/overview/terms-and-privacy-policy-flow/).
+
+## 1.1.1
+* Fixes release build configuration for the iOS plugin [#50](https://github.com/AppLovin/AppLovin-MAX-Godot/pull/50)
+* Removed rewarded interstitial references in the Android plugin [#51](https://github.com/AppLovin/AppLovin-MAX-Godot/pull/51)
+
+## 1.1.0
+* Support the latest AppLovinSDK major version release, 13.0.0
```

Comment:
Maybe copy how we do it on the React Native and Flutter changelogs?

Reply:
This is not about the SDK version update since godot doesn't have a specific SDK version in the package.  I guess this is about the overall update for 13.0.0, so we can leave it?

Reply:
Yeah that's fine, just add a period then